data/campaigns/Descent_Into_Darkness/scenarios/01_Saving_Parthyn.cfg

```diff
@@ -16,30 +16,49 @@
+            # po: Dela: little more of a minor character, she's what you might expect from a sister. She isn't wholly against Malin until he kills Drogan, but she also doesn't really want to do much with a necromancer either.
+            # po: Malin: rash and impulsive, he cares a lot about his home, but doesn't see that power itself can be a corrupting force (though he does not seek power just for the sake of it, it is for another purpose which is
+            # po:           to drive off the orcs). He's a little more informal in his speech, meaning more contractions and colloquialisms, and this is intentional.
+            # po: Things to be careful about while editing: there are some poems in S8 and S11, please be mindful when making wording changes there. Some commas are added for readability even though they may not be grammatically correct.
+            # po:
             # po: If the target language has a specialized term for a fortified town, like Latin "oppidum", use it for Parthyn.
-            story= _ "Malin Keshar was born ten years after the death of Haldric IV. He grew up in the northern border town of Parthyn, the second child and eldest son of the baron of the city. Every summer when the mountain passes became clear, orcs came down from the north to prove themselves in battle. Every year Parthyn’s guards repelled the raids, though some of the townsfolk were always lost. Malin’s father led raiding parties to disrupt the orc encampments and send the orcs back to the north."
+            story= _ "Malin Keshar was born ten years after the death of Haldric IV. He grew up in the northern border town of Parthyn, the second child and eldest son of the city’s baron. Every summer, by the time the mountainous paths and high passes shed their wintry gowns of snow, orcs descended from the northern hills to ravage the settlements at the frontier. Every year, Malin’s father led the townsfolk to repel the raids and force the orcs to retreat back to the north."
             {STORYTXT_BACKGROUND_BIGMAP_DID}
         [/part]
         [part]
-            story= _ "When Malin was 10, a mage who passed through the town sensed strong magical abilities in the boy. His parents, seeing a chance for him to live away from the constant battles with the orcs, arranged for him to be sent to the mage community on the Isle of Alduin. There he studied for eight years, while back at home the orc raids increased in their frequency and ferocity. Before Malin could finish his training, however, he was expelled."
+            story= _ "When Malin was ten, a passing mage sensed strong magical abilities in the boy and suggested he be sent to the Isle of Alduin for training in the magical arts. Malin studied there for eight years, learning all manner of magic from basic control over the elements, to the mystic arts and scrying, to alchemy and simple medicine."
```

Comment:
In this sentence, I propose to add a colon after the word "magic". What do others think?

> Malin studied there for eight years, learning all manner of magic from basic control over the elements, to the mystic arts and scrying, to alchemy and simple medicine.

I did review this PR a long time ago but this sentence was changed yesterday just before the merge, that's why I didn't suggest this sooner.

Reply:
For some reason I thought you were suggesting a _semi_-colon and I was hesitant at first. I think a colon is suitable here.

Reply:
I forgot to put a comment here, but I discussed with josteph over Discord already. I think the current wording is okay, but using the colon format would be fine as well (no big preference either way). I don't intend to change it, but if someone else strongly prefers the colon punctuation, please feel free to change it.

Reply:
No worries - I should have mentioned that it works either way but after reading a few times I think I favour the colon. Fine with leaving it as it is, though.